### PR TITLE
example: Add client example

### LINKED ISSUE #41: Need to implement an example of client
## Feature Request

**Is your feature request related to a problem? Please describe.**  

In order to make it easier to understand how to use the library, we should implement an actual usage example

**Describe the solution you'd like**  

We need to implement an example in the client that send information using the library.

**Describe alternatives you've considered**  

**Additional context**

### PR DESCRIPTION
**Pull request by HyeockJinKim**
# Summary 

- It closes #41

## How to Review? 

- Please refer to my self-review submitted as the review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,99 @@ yarn add @0auth/client     // client library
 
 ## Usage
 
-should be written later.
+### In Client
```
Comment (HyeockJinKim) [inline README.md:19]: Add client, server example to README

File: examples/client/.env [hunk 2]
```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```
Comment (HyeockJinKim) [inline examples/client/.env:1]: Setting to not apply eslint in the root folder.

File: examples/client/.gitignore [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (HyeockJinKim) [inline examples/client/.gitignore:1]: The files below are the files resulting from CRA.

File: examples/client/src/App.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        'Content-Type': 'application/json'
+      },
+      method: 'POST',
+      body: JSON.stringify({ properties }),
+    }).then(res => res.json() as unknown as Signature)
+      .then(res => {
+        storeSignature(properties, res, StorageType.LocalStorage);
+        setSign(res)
+      });
+  };
```
Comment (HyeockJinKim) [inline examples/client/src/App.tsx:37]: Send user information and receive authentication from the server.
Store properties and signature using `0auth`library and setSign for use in react component.

File: examples/client/src/component/movies.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        'Content-Type': 'application/json'
+      },
+      method: 'POST',
+      body: JSON.stringify({ properties, sign }),
+    }).then(res => res.json() as unknown as Ticket)
+      .then(res => {
+        movieList[id].ticket = res;
+        setMovieList([...movieList]);
+      });
+  };
```
Comment (HyeockJinKim) [inline examples/client/src/component/movies.tsx:33]: Perform movie reservations through authenticated user information.

File: examples/server/src/index.ts [hunk 6]
```diff
@@ -13,18 +16,19 @@ const mockMovies = [
...
 const privateKey = {
   key: '2ef40452ec154cd38efdc8ffa52e7f513f7d2b2a77e028342bde96c369e4f77a',
   type: KeyType.ECDSA,
 };
 
-const publicKey = {
-  key: '048da7b63430eb4db203177baf2e8699a25116561624e67a31c2bf288d54216ce3f6f9c7b81fdbb5732342475a6ee5ccab883277ddbb38fdb79ab5424d401b844a',
-  type: KeyType.ECDSA,
-};
+const publicKey = publicKeyFromSecret(privateKey);
```
Comment (HyeockJinKim) [inline examples/server/src/index.ts:27]: refactoring

File: examples/server/src/index.ts [hunk 7]
```diff
@@ -13,18 +16,19 @@ const mockMovies = [
...
+
+app.get('/movies', (req, res) => {
+  res.send(mockMovies);
+});
```
Comment (HyeockJinKim) [inline examples/server/src/index.ts:31]: for use in client.

File: lerna.json [hunk 8]
```diff
@@ -2,7 +2,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "1.0.7",
+  "version": "1.0.8",
```
Comment (HyeockJinKim) [inline lerna.json:5]: Library version update was performed in npm.

**Review by HyeockJinKim — COMMENTED** [review]
self review

**Review by anthonyminyungi — APPROVED** [review]
There are too many files that don't need to review at all.
I found  the example doesn't work properly, but because you said it works fine for you,
I'll merge it.

**Pull request merged**